src/server/utils/logging/fastifyPlugin.js

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2019 American Express Travel Related Services Company, Inc.
```

Comment:
new file so should be this year.
```suggestion
 * Copyright 2022 American Express Travel Related Services Company, Inc.
```